data/110/873/951/3/1108739513.geojson

```diff
@@ -220,8 +220,10 @@
     ],
     "wof:breaches":[],
     "wof:concordances":{
-        "hasc:id":"JP.HK.HK"
+        "hasc:id":"JP.HK.HK",
+        "meso:local_id":"01202"
```

Comment:
What's the story with the meso concordances and `local_id` naming convention? I don't see a key in the meso source json and don't recall storing meso properties directly in the concordances field..

Reply:
There are _some_ MESO countries with `meso:local_id` and `meso:id` properties, but they are not universal in the AOTM deliveries. 

Looking back thru more of the MESO deliveries (as stored as properties on each country's `county` features and comparing them with known "official" concordances for those countries, there is _often_ useful information to promote to the WOF record's concordance hash list.

In a perfect world we'd research and create a new source for that country's national mapping agency. But between MESO and QA there are a couple dozen and that's a rainy day future project.

- MESO countries: BY, CN, JP, KR, QA, VN, and a handful of other countries / partial gap coverage.
- QS countries: 
    - BG regions
    - CY regions
    - CZ regions
    - GL regions
    - HR regions
    - HU regions
    - IE regions
    - IE county
    - LT regions
    - LV regions
    - MD regions
    - MT regions
    - PT regions
    - RO regions
    - RS regions
    - SK regions
    - UA regions
    - LT county
    - SK county
    - UA county

I've recorded this as a comment in this PR's linked issue... and now filed it as a separate followup task in https://github.com/whosonfirst-data/whosonfirst-data/issues/2166.

Reply:
It's also likely the `meso:local_id` is missing the region code prefixes, but without additional research it's hard to know. So adding this concordance under `meso` namespace also indicates some skepticism is warranted during use.